router/router.go

```diff
@@ -663,10 +663,28 @@ func (worker *workerT) processDestinationJobs() {
 								if worker.rt.transformerProxy {
 									jobID := destinationJob.JobMetadataArray[0].JobID
 									pkgLogger.Debugf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Request started`, worker.rt.destName, jobID)
+
+									// setting metadata
+									firstJobMetadata := destinationJob.JobMetadataArray[0]
+									proxyReqMetadata := transformer.ProxyRequestMetadata{
+										SourceID:      firstJobMetadata.SourceID,
+										DestinationID: firstJobMetadata.DestinationID,
+										WorkspaceID:   firstJobMetadata.WorkspaceID,
+										JobID:         firstJobMetadata.JobID,
+										AttemptNum:    firstJobMetadata.AttemptNum,
+										DestInfo:      firstJobMetadata.DestInfo,
+										Secret:        firstJobMetadata.Secret,
+									}
+									// forming the proxy request payload
+									proxyReqPayload := transformer.ProxyRequestPayload{
+										PostParametersT: &result[resultInd],
```

Comment:
another reason why we wouldn't need a pointer?

Reply:
Done